src/containers/Article/index.js

```diff
@@ -61,6 +65,14 @@ export class ArticleForm extends Component {
     this.setState({ body: blocks });
   };
 
+  handleTags = (newValue) => {
```

Comment:
what should `newValue` contain? please make it more descriptive

Reply:
On it